### ISSUE TITLE
Allow manually specify java runtime per project

### ISSUE THREAD

**Issue by Yanpas** [redhat-developer/vscode-java#2151]
[provide a description of the issue]

##### Environment
- Operating System: arch linux
- JDK version: 16
- Visual Studio Code version: 1.60
- Java extension version: 0.82

There is `java.configuration.runtimes` with `default:true` option to select  default runtime. However I have a gradle project where extension tries to use JDK 16, instead of 8. `java.configuration.runtimes` are not supported to be set locally

**Comment by Yanpas** [comment]
I see the cause. Anyway, do you find it reasonable to let the user override autoguessing which is not always working? What if target version is 7 for example, but I have only 8 and 11 jdks?

**Comment by mshindal** [comment]
Agreed, we should be able to set the Java version for each project manually. Today, if someone doesn't use one of the officially supported build systems (Gradle or Maven) - they're out of luck.
